### PR TITLE
fix cursor ends up in wrong location #2353

### PR DESCRIPTION
**Pull request by kpaharelau**
It is very strange but Robot mono font doesn't work correctly with codemirror editor. I have changed robot mono on ubuntu mono font.

### CODE REVIEW

File: core/src/main/web/app/styles/vendor_overrides.scss [hunk 1]
```diff
@@ -26,6 +26,7 @@
 
 .CodeMirror pre {
   z-index: initial;
+  font-size: 14px;
```
Comment (scottdraves) [inline core/src/main/web/app/styles/vendor_overrides.scss:29]: why is this needed?

**Comment by kpaharelau** [comment]
Font ubuntu-mono working good, but it looks small for default size

image

image

image

**Comment by scottdraves** [comment]
OK this looks good.  Please remove the Roboto Mono font, and everywhere it appeared replace it with Ubuntu Mono.

**Comment by scottdraves** [comment]
:+1:
